kerastuner/tuners/bayesian.py

```diff
@@ -32,6 +33,8 @@ class BayesianOptimizationOracle(oracle_module.Oracle):
             as initial training data for Bayesian optimization. If not specified,
             a value of 3 times the dimensionality of the hyperparameter space is
             used.
+        acq: String. Acquisition function. 'ucb' for Upper Confidence Bound
```

Comment:
I think it's better to name this `acquisition_function`. We should accept the strings "ucb" and "ei", and also accept any subclass of ```AcquisitionFunction```